### PR TITLE
feat: filter and filter reset added

### PR DESCRIPTION
**Pull request by Vivek-Py**
## Feature
Filters added and an option to reset the filters

### CODE REVIEW

File: src/containers/Home/index.css [hunk 1]
```diff
@@ -1,5 +1,10 @@
 aside.sidebar {
   align-self: flex-start;
+  position: sticky;
+  top: 0;
+
+  box-shadow: rgba(0, 0, 0, 0.05) 0px 6px 24px 0px,
```
Comment (aurobindoGupta) [inline src/containers/Home/index.css:6]: have a diff file for colors

File: src/containers/Home/index.js [hunk 2]
```diff
@@ -1,78 +1,155 @@
...
+          setGenderFilter([...genderFilter, e.target.value]);
+        else
+          setGenderFilter([
+            ...genderFilter.filter((item) => item !== e.target.value),
+          ]);
+      };
+    } else if (type === "rating") {
+      return function (e) {
+        if (e.target.checked)
+          setStarFilter([...new Set([...starFilter, +e.target.value[0]])]);
```
Comment (aurobindoGupta) [inline src/containers/Home/index.js:29]: nice , what is this, what does this do, the new Set part. DM me.

**Review by aurobindoGupta — APPROVED** [review]
nice work.
clean code.
clear distinguished components

File: src/containers/Home/index.css (hunk 1 above)
Comment (Vivek-Py) [reply src/containers/Home/index.css:6]: Sure thing

File: src/containers/Home/index.js (hunk 2 above)
Comment (Vivek-Py) [reply src/containers/Home/index.js:29]: Done, shared the info

**Pull request merged**